syft/frameworks/torch/tensors/interpreters/autograd.py

```diff
@@ -265,3 +265,60 @@ def float_precision(self):
         if isinstance(self.child, torch.Tensor) and not self.child.is_wrapper:
             return self.child
         return self
+
+    @staticmethod
+    def simplify(tensor: "AutogradTensor") -> tuple:
+        """Takes the attributes of an AutogradTensor and saves them in a tuple.
+            Or simply said, it serializes an AutogradTensor
+        Args:
+            tensor: an AutogradTensor.
+
+        Returns:
+            tuple: a tuple holding the unique attributes of the AutogradTensor.
+        """
+        chain = None
+        # send the child over
+        if hasattr(tensor, "child"):
+            chain = syft.serde._simplify(tensor.child)
```

Comment:
Could be contracted into a ternary operator:
chain = syft.serde._simplify(tensor.child) if hasattr(tensor, "child") else None

Reply:
Done!